npm-shrinkwrap.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "ass": {
       "version": "1.0.0",
-      "from": "ass@git://github.com/jrgm/ass.git#5be99ee",
+      "from": "../../../../../var/folders/rv/6mf4_t5x169fdxtrnd14y7500000gq/T/npm-84113-846bd3c1/git-cache-e6b51c1266b7/5be99ee7abc9fcf63f9ebcc37b151b9c822146d1",
```

Comment:
This may be one of the most horrific `from` paths I've ever seen.
Can't wait to try switching to [**npm-shrinkwrap**](https://github.com/uber/npm-shrinkwrap) so we can just remove these `from` fields and clean up these diffs.